### PR TITLE
FHAC-648: Soap Test to Validate Audit Query Log Service

### PR DESCRIPTION
**Pull request by sailajaa**
1. Created Validate Audit Query Log service Test under Bimodal folder to test Audit query service.

### CODE REVIEW

**Comment by alameluchidambaram** [comment]
As we discussed it would be good if we add negative test that search query parameters didn't retrieve any results and still the response is Schema complaint

**Comment by sailajaa** [comment]
@alameluchidambaram : Updated script with element namespace for xpath and included 2 new testcases. Ready for review and test

File: Product/SoapUI_Test/RegressionSuite/Bimodal/ValidateAuditQueryLogService/ValidateAuditQueryLogService-soapui-project.xml [hunk 1]
```diff
@@ -0,0 +1,1010 @@
...
+         <EventBeginDate>?</EventBeginDate>
+         <!--Optional:-->
+         <EventEndDate>?</EventEndDate>
+         <!--Optional:-->
+         <RemoteHcidList>
+            <!--Zero or more repetitions:-->
+            <RemoteHcid>?</RemoteHcid>
+         </RemoteHcidList>
+      </urn:QueryAuditEventsRequestType>
+   </soap:Body>
```
Comment (alameluchidambaram) [inline Product/SoapUI_Test/RegressionSuite/Bimodal/ValidateAuditQueryLogService/ValidateAuditQueryLogService-soapui-project.xml]: Can you remove these unused test cases? I can see for all 3 services.

**Comment by alameluchidambaram** [comment]
Reviewed this PR and good with merge.

**Pull request merged**